lib/daru.rb

```diff
@@ -95,6 +95,7 @@ def plotting_library= lib
 require 'forwardable'
 require 'erb'
 require 'date'
+require 'mechanize'
```

Comment:
Please, when implementing new features, take a time to look around and understand, what approaches are taken on similar features implementations. For this task, you can look at `has_nyaplot?` and similar things in codebase.